engine/core/IgeSpriteSheet.ts

```diff
@@ -15,7 +15,7 @@ export class IgeSpriteSheet extends IgeTexture {
 	IgeSpriteSheet = true;
 	_cells: IgeTextureCellArray = [];
 	_spriteCells: IgeTextureCellArray = [];
-	_sheetImage?: IgeImage | IgeCanvas;
+	_sheetImage?: IgeImage;
```

Comment:
Do we want to remove support for canvas based images here?

Reply:
See above